azure/templates/smoke-tests.yml

```diff
@@ -24,7 +24,7 @@ steps:
               export BASE_URL="${{ parameters.base_url }}"
               export ROOT_PATH="$(SERVICE_BASE_PATH)"
               export CLIENT_ID="$(CLIENT_ID)"
-              export JWT_RS256_PRIVATE_KEY=$(echo "$(JWT_RS256_PRIVATE_KEY)" | base64 -d | sed -z 's/\n/\\n/g')
+              export TOKEN_URL=https://${{ parameters.environment }}.api.service.nhs.uk/oauth2/token
```

Comment:
What is the address if this is prod? There is a period in there....

Reply:
@JalfResi I put an if condition in ther